apps/editor.planx.uk/src/lib/api/gis/requests.ts

```diff
@@ -0,0 +1,39 @@
+import type { GISResponse } from "@opensystemslab/planx-core/types";
+
+import apiClient from "../client";
+
+export const getTeamGISData = async ({
+  teamSlug,
+  geom,
+  dataValues,
+}: {
+  teamSlug: string;
+  geom: string;
+  dataValues: string[];
+}): Promise<GISResponse & { url: string }> => {
+  // Get current query parameters (eg ?analytics=false&sessionId=XXX) to determine if we should audit this response
+  const urlSearchParams = new URLSearchParams(window.location.search);
+  const params = Object.fromEntries(urlSearchParams.entries());
+
+  const { data, request } = await apiClient.get(`/gis/${teamSlug}`, {
+    params: {
+      geom,
+      vals: dataValues.join(","),
+      ...params,
+    },
+  });
+
+  // Return full URL for auditing purposes
+  return { ...data, url: request.responseURL };
```

Comment:
We could (should?) consider returning the `url` from the API as it's always required for auditing, but we can also get it reliably this way.